Library/Formula/libmill.rb

```diff
@@ -0,0 +1,29 @@
+class Libmill < Formula
+  desc "Go-style concurrency in C"
+  homepage "http://libmill.org/"
+  head "https://github.com/sustrik/libmill.git"
+  url "http://libmill.org/libmill-0.16-beta.tar.gz"
+  sha256 'c6d68d8291dadf2ee111f6ceed057f2fd267e7dc334bef8d15b6316c35b6a0ca'
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "cmake" => :build
+  env :std
```

Comment:
the superenv causes the injection of the CFLAG '-O2' which provokes an optimization bug in libmill.

Setting env :std allows libmill to successfully set it's own CFLAGS on detection of Clang. 

I could not make it build without env :std. 

Perhaps someone who better understands homebrew can figure out what magic is needed to keep it from setting anything other than -O1 as a compilation flag.